packages/fxa-content-server/server/lib/beta-settings.js

```diff
@@ -77,6 +77,10 @@ const settingsConfig = {
   glean: { ...config.get('glean'), appDisplayVersion: config.get('version') },
   redirectAllowlist: config.get('redirect_check.allow_list'),
   sendFxAStatusOnSettings: config.get('featureFlags.sendFxAStatusOnSettings'),
+  showReactApp: {
+    signUpRoutes: config.get('showReactApp.signUpRoutes'),
+    signInRoutes: config.get('showReactApp.signInRoutes'),
```

Comment:
One quick note about this.

I added this to conditionally redirect to React or Backbone between SignIn/SignUp. In the Signup container, I used this accordingly. 

However, I didn't adjust the SignIn container to account for if SignUp routes were on or not because I didn't think it was worth the effort since SignUp should be rolled out when we roll SignIn out, but I left the config around (which I could remove). This just means **if we ever don't have React sign up rolled out but we _do_ have React sign in rolled out**, we're going to want a 1-pointer to also account for this in the signin container.